gm4_heart_canisters/data/heart_canisters/functions/apply_health_boost.mcfunction

```diff
@@ -0,0 +1,11 @@
+effect clear @s health_boost
```

Comment:
Do we need to clear the heath boost here? Won't it just be overridden if the health boost already on the player isn't stronger than the one to be applied?

Reply:
It needs to be cleared so it can correctly lower the health boost when the player removes a heart canister